### PR TITLE
Pil 1098

### PR DESCRIPTION
**Pull request by olakunlekuye2**
Added the Knock-back page BTN6.1 suggests to the user to complete a UKTR nil return and directs them to the page for directing FM or Agent to Third Party submission of UKTR.

### CODE REVIEW

File: app/views/btn/BtnSubmitUKTRView.scala.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+@*
```
Comment (tev162) [inline app/views/btn/BtnSubmitUKTRView.scala.html:1]: Is this page being used somewhere?

File: conf/messages.en [hunk 2]
```diff
@@ -212,3 +212,13 @@ btnRevenues750InNext2AccountingPeriods.checkYourAnswersLabel = btnRevenues750InN
...
+btn.btnSubmitUKTR.title = Based on your answer, your group must submit a UK Tax Return
+btn.btnSubmitUKTR.heading = Based on your answer, your group must submit a UK Tax Return
+btn.btnSubmitUKTR.p1 = Based on your answer, you cannot submit a Below-Threshold Notification as the group is expected to make consolidated annual revenues of €750 million or more within the next 2 accounting periods.
+btn.btnSubmitUKTR.p2 = You can submit a nil return UK Tax Return if your group is not due to pay any Domestic Top-up Tax for their current accounting period.
+btn.btnSubmitUKTR.link = Find out more about submitting a late or new UK Tax Return
+
+
+
+
+
```
Comment (tev162) [inline conf/messages.en:224]: minor: you can remove these extra lines

File: app/views/btn/BtnSubmitUKTRView.scala.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ * Copyright 2024 HM Revenue & Customs
```
Comment (tev162) [inline app/views/btn/BtnSubmitUKTRView.scala.html:2]: just need to add a ViewSpec

Comment (olakunlekuye2) [reply app/views/btn/BtnSubmitUKTRView.scala.html:2]: Okay

File: app/views/btn/BtnSubmitUKTRView.scala.html (hunk 1 above)
Comment (olakunlekuye2) [reply app/views/btn/BtnSubmitUKTRView.scala.html:1]: oops sorry , done now

File: conf/messages.en (hunk 2 above)
Comment (olakunlekuye2) [reply conf/messages.en]: Done

**Review by tev162 — APPROVED** [review]

**Review by ridow-hmrc — APPROVED** [review]

**Pull request merged**